### PR TITLE
Switch to repo.spring.io

### PR DESCRIPTION
**Pull request by brianbolt**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,36 +57,38 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 	</scm>
 	<repositories>
 		<repository>
-			<id>spring-maven-release</id>
-			<name>Spring Maven Release Repository</name>
-			<url>https://maven.springframework.org/release</url>
+			<id>spring-snapshots</id>
+			<name>Spring Snapshots</name>
+			<url>https://repo.spring.io/snapshot</url>
```
Comment (bffrost) [inline pom.xml:62]: @brianbolt Do we want snapshot rather than release? Not sure why this was switched

Comment (brianbolt) [reply pom.xml:62]: Yeah, we do want release. I copy pasted this and missed that it changed.

Comment (brianbolt) [reply pom.xml:62]: I've reverted this for now and will come back to it later since the old builds are working

Comment (bffrost) [reply pom.xml:62]: Sounds good, thanks

**Pull request merged**